app/models/translations.rb

```diff
@@ -4,12 +4,14 @@
 # OpenSSF Best Practices badge contributors
 # SPDX-License-Identifier: MIT
 
-class Translations
+module Translations
+  module_function
+
   # This class is a used to gather sets of translations which will commonly
   # be called in bulk.  For example, there are translations we would like
   # to export to JavaScript.  We can use a method here to do that quickly.
 
-  def self.for_js
+  def for_js
```

Comment:
You sure about this switch from class to module, and removing "self."? What rubocop rule is recommending this?

Reply:
This works due to the addition of module_function on line 8 and changing the class to a module. There's a discussion about it https://github.com/rubocop/ruby-style-guide/issues/556 here and I believe we could ignore https://docs.rubocop.org/rubocop/cops_style.html#stylemodulefunction Style/ModuleFunction if we don't care about or want this.

Reply:
this function seems to be tested pretty thoroughly here https://github.com/coreinfrastructure/best-practices-badge/blob/main/test/models/translations_test.rb

Reply:
[AllowOnSelfClass is the other thing to look at here](https://docs.rubocop.org/rubocop/cops_style.html#stylestaticclass) is the other one to look at here